[Speed]Update how-do-i-enable-http2-server-push-in-wordpress.mdx (#18158)
cloudflare/cloudflare-docs: src/content/docs/support/third-party-software/content-management-system-cms/how-do-i-enable-http2-server-push-in-wordpress.mdx

Cloudflare supports HTTP/2 Server Push and it can be enabled for stylesheets and scripts using Cloudflare’s WordPress plugin. In order to utilise this feature, you must first ensure you have the Cloudflare WordPress plugin [installed and set-up on your site](https://support.cloudflare.com/hc/en-us/articles/227634427-Using-Cloudflare-with-WordPress).

> Once the plugin is installed, you can enable HTTP/2 Server Push simply by adding the following configuration code to your `wp-config.php` file:

Reviewer: ```suggestion
Once the plugin is installed, you can enable HTTP/2 Server Push by adding the following line to your `wp-config.php` file:
```

```php

> define('CLOUDFLARE\_HTTP2\_SERVER\_PUSH\_ACTIVE', true);

Reviewer: ```suggestion
define('CLOUDFLARE_HTTP2_SERVER_PUSH_ACTIVE', true);
```
Per Zein's suggestion.

```

> You should insert this line above where it says *"/\* That's all, stop editing! Happy blogging. \*/*", like follows:|

Reviewer: ```suggestion

You should insert this line above where it says *"/\* That's all, stop editing! Happy blogging. \*/*", like follows:|
```
